ocs_ci/ocs/resources/cloud_manager.py

```diff
@@ -125,8 +124,25 @@ def delete_uls(self, name):
         the appropriate implementation
 
         """
+        if self.verify_uls_exists(name) is False:
+            logger.warning(
+                f"Underlying Storage {name} does not exist, and was not deleted."
```

Comment:
Could you please rephrase the warning to maybe `Underlying Storage {name} does not exist, hence it can't be deleted`